fluent-syntax/src/serializer.ts

```diff
@@ -136,19 +136,29 @@ function serializeTerm(term: AST.Term): string {
 
 
 function serializeAttribute(attribute: AST.Attribute): string {
-  const value = indent(serializePattern(attribute.value));
+  const value = indentExceptFirstLine(serializePattern(attribute.value));
   return `\n    .${attribute.id.name} =${value}`;
 }
 
 
 function serializePattern(pattern: AST.Pattern): string {
   const content = pattern.elements.map(serializeElement).join("");
-  const startOnNewLine =
+  const maybeStartOnNewLine =
     pattern.elements.some(isSelectExpr) ||
     pattern.elements.some(includesNewLine);
 
-  if (startOnNewLine) {
-    return `\n    ${indent(content)}`;
+  if (maybeStartOnNewLine) {
+    const firstElement = pattern.elements[0];
```

Comment:
I'd prefer this code to be a function like the other two.

Reply:
I used to have it in a separate function which returned a boolean, but I then inlined it here because the `false` case resulted in two outcomes: `indentExceptFirstLine(content)` in line 157 and `content` in 164, and I didn't want to add any special sentinel return values.

However, when `content` is on a single line, `indentExceptFirstLine(content)` is a no-op, so I can also use it in line 164.